src/sources/kafka.rs

```diff
@@ -56,6 +56,14 @@ enum BuildError {
     KafkaSubscribeError { source: rdkafka::error::KafkaError },
 }
 
+/// Metrics configuration.
+#[configurable_component]
+#[derive(Clone, Debug, Default)]
+struct Metrics {
+    /// Expose topic lag metrics.
```

Comment:
Can we add information about the new metrics we'll be creating here? Names, tags, types, when this should be used, etc. 

Just by looking at the docs, it's hard to tell what the flag does.

Reply:
Yep, we definitely can and should. Could you please suggest to me the right place for placing the information about these metrics? I guess it should be somewhere in `kafka` source CUE file near `metrics` section, but I am not sure.

Reply:
You would add an entry here: https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/sources/kafka.cue#L110

Then you would add specific info about the metric here (this is what the reference above is for): https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/sources/internal_metrics.cue#L618

Then, we should also mention inline here (after the `///`) the names of the metrics that are created as a result of enabling the flag.

Reply:
Fixed.